tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -1,5 +1,5 @@
 <?php
```

Comment:
It's not clear to me that we need this test any more, given that it is basically testing the same functionality that we already assert on the jetpack sync client.